.github/workflows/latest-release.yml

```
@@ -0,0 +1,118 @@
+name: Automated Latest Release
+
+env:
+  CHANNEL: 'latest'
+  PRERELEASE: 'true'
+
+on:
+  pull_request:
+    types: [closed]
+    branches:
+      - main
+
+# Only cancel in-progress jobs or runs for the current workflow run
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  latest_release:
+    if: github.event.pull_request.merged == true && github.base_ref == 'main' && !contains(github.event.pull_request.labels.*.name, 'skip release')
+    runs-on: ubuntu-22.04
+    steps:
```

I'd suggest checking if the PR was merged. The action can fire for closed and unmerged PRs as well.

```suggestion
    if: github.event.pull_request.merged == true
    steps:
```

---

Done